lib/utils/report.js

```diff
@@ -109,3 +109,26 @@ module.exports = function report(problem) {
 
 	result.warn(warningMessage, warningProperties);
 };
+
+/**
+ * @param {import('postcss').Node | undefined} node
+ * @param {number | undefined} index
+ * @param {number | undefined} endIndex
+ * @returns {number | undefined}
+ */
+function getStartLine(node, index, endIndex) {
+	if (node === undefined) return undefined;
```

Comment:
```suggestion
	if (!node) return;
```

Maybe a person taste here, or do we have a lint rule to enforce a consistent style?

Reply:
I have no strong opinion about this style. 😅